tools/wptrunner/wptrunner/browsers/chrome_android.py

```diff
@@ -68,7 +68,7 @@ def env_extras(**kwargs):
 
 
 def env_options():
-    return {"bind_hostname": "true"}
+    return {"bind_hostname": True}
```

Comment:
Let's just make `True` the default and remove all of these

Reply:
done